src/Configuration.php

```diff
@@ -121,7 +126,21 @@ public function getPath()
      */
     public function disableColors()
     {
-        return $this->write('colorsEnabled', false);
+        if ( $this->colorsEnableExplicit ) {
+            return $this;
+        }
+        else {
+            return $this->write('colorsEnabled', false);
+        }
+    }
+
+    /**
+     * Force output colors even without TTY support.
+     *
+     * @return $this
+     */
+    public function enableColorsExplicit() {
```

Comment:
surprised scrutinizer didn't catch this (I might have to tweak a config). For methods we try to put the opening bracket on a new line i.e:

```php
public function enableColorsExplicit()
{
}
```
And for the body of the code, I think I would prefer using the fluent interface of `write`:

```php
public function enableColorsExplicit()
{
    return $this
        ->write('colorsEnableExplicit', true)
        ->write('colorsEnabled', true);
}
```